documentation/Get-PnPPlannerTask.md

```diff
@@ -89,7 +95,7 @@ Accept wildcard characters: False
 ```
 
 ### -Group
-Specify the group id of group owning the plan.
+Specify the group idor name of group owning the plan.
```

Comment:
Space missing :-)

Reply:
Fixed :-)

Can you restart the workflow to fix the commit for Set-PnPPlannerTask please?